cli/rbd/group.py

```diff
@@ -0,0 +1,256 @@
+from copy import deepcopy
+
+from cli import Cli
+from cli.utilities.utils import build_cmd_from_args
+
+
+class Group(Cli):
```

Comment:
please cover all these test operations in the existing test.

Reply:
as we discussed we are planning to automate only 2 usecases, not all functional combinations. So let this be there for 2 customer usecases. SInce not all customers may not do group snapshot, I added "group image add" and "group snap" to loosely decouple

1. group image add
2. group snap